crates/algorithms/src/haversine_formula/mod.rs

```diff
@@ -0,0 +1,81 @@
+use crate::PI;
```

Comment:
This will not be required once you use a Floating trait, as they will have an inlined function that returns this